volmdlr/edges.py

```diff
@@ -247,6 +245,10 @@ class Line2D(Line):
     """
 
     def __init__(self, point1, point2, *, name=''):
+        if isinstance(point1, list):
+            point1 = volmdlr.Point2D(point1[0], point1[1])
+        if isinstance(point2, list):
```

Comment:
Same as previous comment : 

```suggestion
        if dc.is_sequence(point2):
```